ingestfile

```diff
@@ -84,10 +94,26 @@ else
     _report -w "you're in a directory that doesn't actually exist. The script exited and you need to cd into a different directory. ¯\_(ツ)_/¯ "
     exit 1
 fi
+
+if [ "${SILENT_RUNNING}" == "Y" ] && [[ "${OP}" == "" || "${INPUT}" == "" || "${MEDIAID}" == "" ]] ; then
+    echo "You must specify each of: operator name, input filepath, media id. Please assign a value to each of these and try again. :)"
+    _report -wt "You tried to use ingestfile in SILENT_RUNNING mode and failed to specify one of the necessary variables: Operator name, input file path, and media id."
+    exit 1
+fi
+
+if [ "${SILENT_RUNNING}" == "Y" ] ; then
+    EDIT_MODE="N"
+    DELIVERYTOOMNEON="N"
+    CLEANUP_CHOICE="Y"
+    PRIORITY="start now"
+    SLATE="N"
+    MAKEACCESSCOPIES="Y"
+fi
+
 [ -z "${OUTDIR_INGESTFILE}" ] && { echo "The processing directory must be set. Use [ -p /path/to/processing/directory ] or run mmconfig to set OUTDIR_INGESTFILE." ; exit 1 ;};
 [ -z "${AIP_STORAGE}" ] && { echo "The AIP Storage directory must be set. Use [ -a /path/to/AIP/storage/directory ] or run mmconfig to set AIP_STORAGE." ; exit 1 ;};
 [ -z "${PODCASTDELIVER}" ] && { echo "A directory for podcast delivery must be set. Use [ -w /path/to/deliver/podcastfiles] or run mmconfig to set PODCASTDELIVER." ; exit 1 ;};
-[ -z "${YOUTUBEDELIVER}" ] && { echo "A directory for youtube delivery must be set. Use [ -y /path/to/deliver/youtubefiles] or run mmconfig to set YOUTUBEDELIVER." ; exit 1 ;};
+# [ -z "${YOUTUBEDELIVER}" ] && { echo "A directory for youtube delivery must be set. Use [ -y /path/to/deliver/youtubefiles] or run mmconfig to set YOUTUBEDELIVER." ; exit 1 ;};
```

Comment:
This is one of a few local changes that I meant to remove from this branch before pushing. Sorry.